index.html

```diff
@@ -24,15 +24,45 @@ <h1 class="title">Hi! I am Ezra nice to meet you.</h1>
                 If you like what you see and have a project you need coder, don't 
                 husitate to contact me.
             </p>
-            <h3 class="links">LET'S CONNECT</h3>
+        </main>
+    </section>
+    <section class="work-page">
+        <img src="images/works/portfolio1.svg" alt="work1" width="100%">
+        <h2 class="work-title">Tonic</h2>
+        <ul class="clients">
+            <li>CANOPY</li>
+            <img src="images/works/Counter.svg" alt="counter" class="counter">
+            <li>BackEndDev</li>
+            <img src="images/works/Counter.svg" alt="counter" class="counter">
+            <li>2015</li>
+        </ul>
+        <p class="highlights">
+            A daily selection of privately personalized reads
+            no accounts for sign-ups required.
+        </p>
+        <ul class="tags">
+            <li><button>HTML</button></li>
+            <li><button>CSS</button></li>
+            <li><button>Js</button></li>
+        </ul>
+        <button class="project-detail">See <br> Project</button>
+    </section>
+    <section id="about-myself">

```

Comment:
- [ ] Kindly work on the spacing of elements in you `about me` section to match the Figma design.

![image](https://user-images.githubusercontent.com/98453979/198718514-43bf4a56-4e37-46ef-a68d-e44f43ba95e9.png)

### Checkout Figma

![image](https://user-images.githubusercontent.com/98453979/198718640-8a46f050-72ad-41a7-8c86-652718935a1f.png)

- [ ] There are some sections missing in your `about me`  section. Kindly add them.

![image](https://user-images.githubusercontent.com/98453979/198718931-1c0986b8-7838-47c6-8490-26631b7be5c8.png)